Plots/Trajectories/NCL_traj_1.py

```diff
@@ -0,0 +1,97 @@
+"""
+NCL_traj_1.py
+===============
+This script illustrates the following concepts:
+   - Plotting a simple trajectory plot
+   - Plotting multiple trajectories in different colors
+   - Plotting every nth time step in a trajectory
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/traj_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/traj_1_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import xarray as xr
+import geocat.datafiles as gdf
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import matplotlib.pyplot as plt
+import matplotlib.ticker as ticker
+import numpy as np
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get('netcdf_files/traj_data.nc'))
+sdata = ds.get('sdata')
+
+
+###############################################################################
+# Define helper function to plot every fourth timestep:
+
+def plot4thTimestep1(nparrayy, nparrayx):
+
+    for x in range(0, len(nparrayx)):
+
+        # Plot green starting point of each trajectory
+        if x == 0:
+            y, x = nparrayy[x], nparrayx[x]
+            plt.scatter(x, y, color='green', s=4)
+
+        # Plot every fourth timestamp
+        if (x + 1) % 4 == 0:
```

Comment:
Should be **`x`** instead of **`(x + 1)`** as follows:

`if x % 4 == 0:`

Because using `(x + 1)` is resulting in a slightly different plot data than the original NCL plot.